charts/rpe-send-letter-service/Chart.yaml

```diff
@@ -1,12 +1,12 @@
 name: rpe-send-letter-service
 apiVersion: v2
 home: https://github.com/hmcts/send-letter-service
-version: 0.3.16
+version: 0.4.1
```

Comment:
why skipped `0.4.0` ?

Reply:
I decided we should start from 1. Ok, I will fix this